### PR TITLE
Created component with shopping list item

### PR DESCRIPTION
**Pull request by magmat88**

### CODE REVIEW

File: src/app/components/shoppingListItem/shoppingListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import plusBtn from "../../images/plusBtn.svg";
+import './shoppingListItem.scss';
+
+export function ShoppingListItem(): JSX.Element {
+    return (
+        <div className="shopping-list-item">
+            <div className="add-product">
+                <div className="btn-add-product"><img src={plusBtn} alt="Circle button with plus symbol" /></div>
```
Comment (earendillo) [inline src/app/components/shoppingListItem/shoppingListItem.tsx:8]: This is not a usecase for img tag, please check https://stackoverflow.com/a/52605225 which seems suitable here

**Comment by earendillo** [comment]
image
fix the horizontal scroll

File: src/app/components/shoppingListItem/shoppingListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import plusBtn from "../../images/plusBtn.svg";
+import './shoppingListItem.scss';
+
```
[the remark this answers is not in the document]
Comment (earendillo) [reply src/app/components/shoppingListItem/shoppingListItem.tsx:3]: interface should prop for item (name and category) and actions for handlers: onAddProduct and onRemoveProduct

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/shoppingListItem/shoppingListItem.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (magmat88) [reply src/app/components/shoppingListItem/shoppingListItem.tsx:3]: done

File: src/app/components/shoppingListItem/shoppingListItem.tsx (hunk 1 above)
Comment (magmat88) [reply src/app/components/shoppingListItem/shoppingListItem.tsx]: imported svg as component

**Pull request merged**